tests/test_great_expectations.py

```diff
@@ -1029,6 +1029,22 @@ def test_read_table(self):
         assert df["Name"][0] == "Allen, Miss Elisabeth Walton"
         assert isinstance(df, PandasDataset)
 
+    def test_read_feather(self):
+        # pass if available pandas is < 0.20.0 (when read_feather was added)
+        pandas_version = re.match(r"(\d+)\.(\d+)\..+", pd.__version__)
+        if pandas_version is None:
+            raise ValueError("Unrecognized pandas version!")
+        else:
+            pandas_major_version = int(pandas_version.group(1))
+            pandas_minor_version = int(pandas_version.group(2))
+            if pandas_major_version == 0 and pandas_minor_version < 20:
+                return
```

Comment:
Very minor tweak (assuming that this logic needs to remain given the above): instead of `return` here, please change to `pytest.skip("test skipped for unsupported pandas version")`